.spellcheck.yml

```diff
@@ -8,6 +8,40 @@ matrix:
     encoding: utf-8
   pipeline:
   - pyspelling.filters.markdown:
+  - pyspelling.filters.context:
+      context_visible_first: true
+      escapes: '\\[\\`~]'
+      delimiters:
+      # Ignore text between inline back ticks as this is code or hightlight words
+      - open: '(?P<open>`+)'
+        close: '(?P=open)'
+      # Ignore surrounded in <> as in RST it is link
+      - open: '<([A-Za-z0-9-_:.]+)|(https?://[^\\s/$.?#].[^\\s]+|[A-Za-z0-9-_:.]+)'
+        close: '>'
   sources:
   - 'docs/*.rst'
+  - '**/*.md'
+  default_encoding: utf-8
+
+- name: JSON schemas
+  aspell:
+    lang: en
```

Comment:
I think we could add `ignore-case: true`, which would shorten our wordlist.

Reply:
ok will add that as well